### PR TITLE
Fix For Sweetalert

### LINKED ISSUE #271: SweetAlert Node Module Error
**Describe the bug**
On starting the angular GUI server using ```npm run start```, the compiler faces an error during starting the server due to the SweetAlert Module Used.
**To Reproduce**
1. ``` cd gui```
2. ```npm install```
3. ```npm run start```

**Expected behavior**
The Server Must start without any error.

**Why this may have happened according to you**
The SweetAlert node module has this error, please check https://github.com/t4t5/sweetalert/issues/890

**Anything else you were doing before this happened?**

**Screenshots**
image

**Desktop (please complete the following information):**
 - Operating System: Ubuntu
 - OS version: 18.04.3
 
**Additional context we should know**
Please Assign this Issue to me to fix it.

### PR DESCRIPTION
**Pull request by prince-7**
## Status
**READY**

## Description
This PR fixes #271 

## Related PRs

branch | PR
------ | ------

## Todos
- [x] Tests
- [x] Documentation

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

```sh
git pull --prune
git checkout <feature-branch>
```

1. 

## Impacted Areas in Application
List general components of the application that this PR will affect:

*  GUI

### CODE REVIEW

**Comment by prince-7** [comment]
@all-contributors please add @prince-7  for Code

File: gui/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,16269 @@
 {
```
Comment (rejahrehim) [inline gui/package-lock.json:1]: Do we need to commit lock file ?

Comment (prince-7) [reply gui/package-lock.json:1]: not really, I will add this to the  .gitignore file.

Comment (prince-7) [reply gui/package-lock.json:1]: actually, we should https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

**Comment by prince-7** [comment]
@all-contributors please add @prince-7 for Documentation

**Pull request merged**